interface/forms/eye_mag/php/eye_mag_functions.php

```diff
@@ -2369,7 +2404,30 @@ function show_PMSFH_panel($PMSFH, $columns = '1')
         onclick="alter_issue2('0','POS','');" class="disabled_button"><?php echo xlt("None"); ?><br /></span>
         <?php
     }
-
+    
+//<!-- Eye Meds -->
+    echo "<br /><span class='panel_title' title='".xlt("Eye Meds")."'>".xlt("Eye Meds").":</span>";
+    ?><span class="top-right btn-sm" href="#PMH_anchor"
+            onclick="alter_issue2('0','Medication','');" style="text-align:right;font-size:8px;"><?php echo xlt("Add"); ?></span>
+    <br />
+    <?php
+    if ($PMSFH[0]['Eye Meds']) {
+         $i=0;
+        foreach ($PMSFH[0]['Eye Meds'] as $item) {
+             if ( ($item['status'] == "Inactive") || ($item['row_subtype'] != "eye") ){
+                $i++;
+                continue;
+            }
+            echo "<span name='QP_PMH_".attr($item['rowid'])."' href='#PMH_anchor' id='QP_PMH_".attr($item['rowid'])."'
+            onclick=\"alter_issue2('".attr(addslashes($item['rowid']))."','Medication','$i');\">".text($item['title'])."</span><br />";
```

Comment:
more for the crazy attr_js function (and note I removed the surrounding single quotes)(note it is much better than addslashes):
```
onclick=\"alter_issue2(".attr_js($item['rowid']).",'Medication','$i');\">".text($item['title'])."</span><br />";
```

Reply:
this change has not yet been addressed